tests/integrations/backup/mod.rs

```diff
@@ -447,26 +443,36 @@ fn test_backup_raw_meta() {
     let mut checksum = 0;
     let mut total_kvs = 0;
     let mut total_bytes = 0;
-    let mut total_size = 0;
     for f in files {
         checksum ^= f.get_crc64xor();
         total_kvs += f.get_total_kvs();
         total_bytes += f.get_total_bytes();
-        total_size += f.get_size();
     }
-    assert_eq!(total_kvs, key_count + 1);
+    assert_eq!(total_kvs, key_count);
     assert_eq!(total_kvs, admin_total_kvs);
     assert_eq!(total_bytes, admin_total_bytes);
     assert_eq!(checksum, admin_checksum);
-    assert_eq!(total_size, 1611);
+    // assert_eq!(total_size, 1619); // the number changed when kv size change, should not be an test points.
```

Comment:
When does kv size change?

Reply:
With different apiversion, the encode methods are different, which lead to different file size.i think the check of total bytes /total kv count should be enough.